torch/csrc/MemoryFormat.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include <torch/csrc/python_headers.h>
+
+#include <c10/core/MemoryFormat.h>
+
+#include <string>
+
+const int MEMORY_FORMAT_NAME_LEN = 64;
+
+struct THPMemoryFormat {
+  PyObject_HEAD
+  at::MemoryFormat memory_format;
+  char name[MEMORY_FORMAT_NAME_LEN + 1];
```

Comment:
Hmm, I'm guessing this was cargo culted from the other enum code. Do we really need this? I did a quick grep and I only see it being passed to `THPUtils_packString` which seems modestly wasteful.

Reply:
It is nice to have, if user decide to ```print(torch.channels_last)``` or for example print all arguments he is sending to the function.